modules/custom/az_paragraphs/src/Plugin/migrate/process/ParagraphsBehavior.php

```diff
@@ -116,6 +116,9 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
     if (!empty($this->configuration['card_style'])) {
       $behavior['card_style'] = $this->configuration['card_style'];
     }
+    if (!empty($this->configuration['card_clickable'])) {
```

Comment:
Would probably be better to use `isset()` here since it's a boolean value.
```suggestion
    if (isset($this->configuration['card_clickable'])) {
```

Reply:
Agreed on this as well.